docs/install/initial_setup.rst

```diff
@@ -21,60 +64,144 @@ To do the initial setup of your Kolibri facility after the installation, follow
 	  The default language configured for content in your browser preferences might override the language you choose in this step. To ensure that Kolibri displays in the desired language, make sure to configure it as default in the browsers of all the devices that will be used to view Kolibri content.
 
 
-#. **Facility name and type.** A "Facility" is the location where you are installing Kolibri, such as a school, training center, or even your home. 
+#. **Select the Advanced setup** option, recommended for schools, educational programs, organizations, or other group learning settings that will share the use of Kolibri. 
+   
+	  .. figure:: /img/advanced-setup.png
+		   :alt:  
 
-   **Non-formal** type can be used in libraries, orphanages, correctional facilities, youth centers, computer labs and similar, **Formal** one in schools and other formal learning contexts, and **Personal** should work best for parent-child or supplementary individual learning. 
+		   Select Advanced setup for use of Kolibri in group learning environments.			
+   
 
-   Formal and non-formal facilities require a name, while personal ones do not.
+#. **Select the name of the device** where Kolibri server will be running from. Choose a meaningful and recognizable name for the device as it will help you for syncing and importing processes later on. 
```

Comment:
```suggestion
#. **Select the name of the device** on which Kolibri server will be running. Choose a meaningful and recognizable name because it will help you identify it during syncing and importing processes later on. 
```